docs/compose/FieldSet/README.md

```diff
@@ -0,0 +1,43 @@
+# FieldSet
+
+## Installation
+
+Backpack Compose is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-compose). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
+
+FieldSet is a component which wraps its content (user input or form element) and adds optional title, description and
+error labels around it depending on the field status.
+
+The statuses available is Default, Disabled, Error and Validated. The statuses automatically dispatched to
```

Comment:
are, rather than is, dispatch rather than dispatched

Reply:
being dispatched, perhaps?

Reply:
actually: The statuses are automatically dispatched to